### PR TITLE
feat(news): add sightseeing article

### PR DESCRIPTION
**Pull request by robertkowalski**
denn hamburg ist immer einen besuch wert :)

copy of http://2014.jsunconf.eu/news/hamburg-sightseeing-and-tips/

### CODE REVIEW

File: contents/news/hamburg-first-visit/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Tips and Sightseeing for your trip to Hamburg
+date: 2015-01-08
+template: article.jade
+---
+
+First time visiting Hamburg? Having some days left before or after JS Unconf?
+
+[Read our tips and suggestions!](http://2015.jsunconf.eu/news/hamburg-first-visit/)
```
Comment (ro-ka) [inline contents/news/hamburg-first-visit/index.md]: This is linking to itself…

File: contents/news/hamburg-first-visit/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+## Public Transportation
+
+The public transportation in Hamburg is done by the [HVV](http://www.hvv.de/en/index.php). The important thing to know is that you **need to buy a ticket before you go on the platform** and you don't need to clock in the tickets, they are valid after you bought them. Don't go behind the golden lines in the ground without a ticket! For busses you can buy your ticket at the driver.
+
+A "Tagesticket" - a ticket that is valid the whole day and in large parts of Hamburg is the best option most of the time, even if you just take 2 rides. It costs around 7€.
```
Comment (ro-ka) [inline contents/news/hamburg-first-visit/index.md]: Add another `–` after Hamburg.

**Comment by ro-ka** [comment]
:checkered_flag:

**Comment by robertkowalski** [comment]
deployed and twittered. good feedback, thanks @ro-ka

**Comment by ro-ka** [comment]
*Closed with unmerged commits*?

**Comment by robertkowalski** [comment]
@ro-ka yes, as the SHA changes during a rebase github can not detect the change any more.

I try to avoid the big green button and merge fast-forward as the merge commits tend to clutter the history

merged as 85d54b4d78071f14d0f24f8d7663d0dcd07bbea1

**Pull request closed without merging**